R/ChromBackend.R

```diff
@@ -87,41 +117,30 @@
 #'
 #' @param j For `[`: ignored.
 #'
-#' @param msLevel `integer` defining the MS level of the chromatograms to which
-#'     the function should be applied. For `filterMsLevel()`: the MS level to
-#'     which `object` should be subsetted.
-#'
-#' @param mz For `filterMzValues()`: `numeric` with the m/z values of
-#'     chromatograms to keep. All chromatograms with their `mz` chromatogram
-#'     variable matching any of the values provided with this parameter are
-#'     retained. Parameters `ppm` and `tolerance` allow relaxed matching.
-#'     For `filterMzRange()`: `numeric(2)` defining the lower and upper
-#'     boundary of the m/z range. Chromatograms with their `mz` chromatogram
-#'     variable within this range are retained.
-#'
 #' @param name For `$` and `$<-`: the name of the chromatogram variable to
 #'     return or set.
 #'
 #' @param object Object extending `ChromBackend`.
 #'
-#' @param ppm For `filterMzValues()`: m/z-relative acceptable difference (in
-#'     parts-per-million) for m/z values to be considered *matching*.
-#'
-#' @param tolerance For `filterMzValues()`: largest acceptable absolute
-#'     difference in m/z values to consider them *matching*.
-#'
 #' @param value replacement value for `<-` methods. See individual
 #'     method description or expected data type.
 #'
 #' @param x Object extending `ChromBackend`.
 #'
 #' @param ... Additional arguments.
 #'
-#' @section Backend functions:
+#' @section Mandatory methods:
 #'
 #' New backend classes **must** extend the base `ChromBackend` class and
 #' implement the following mandatory methods:
 #'
+#' **Phili: I have now implemented a lot of default methods, so should I maybe
+#'  remove these from the list below ? or keep them and say that if not
```

Comment:
I would, like you did, split the method in to mandatory methods that must be implemented and optional ones with default implementations. You should still list and document the optional methods.